_typos.toml

```diff
@@ -0,0 +1,9 @@
+[default]
```

Comment:
What's with the underscore prefix? Shouldn't it be .typos.toml?

Reply:
This is so that the typos utility treats it as a configuration file.